hooks/paymaster.tsx

```diff
@@ -51,6 +51,8 @@ const usePaymaster = (
   const { isDeployed, deploymentData } = isStarknetDeployed(account?.address);
   const [deploymentTypedData, setDeploymentTypedData] = useState<TypedData>();
   const [invalidTx, setInvalidTx] = useState<boolean>(false);
+  const [txErrorMessage, setTxErrorMessage] = useState<string>();
+  const [txShortErrorMessage, setTxShortErrorMessage] = useState<string>();
 
```

Comment:
I think you can find a better way to manage the error messages than having to setter for it